src/scss/_yfm-only.scss

```diff
@@ -0,0 +1,10 @@
+/**
+  Note: This file excludes cut and tabs for flexibility,
```

Comment:
Plz add info about why we are excluding "cut" and "tabs" - because they are separate extensions (packages). "note", "file", and "table" will also be excluded from this file (and moved to `yfm.scss`) once they are moved to separate packages.